_data/projects/gifbar.yml

```diff
@@ -0,0 +1,9 @@
+name: GifBar
+desc: Find Awesome Gif's right in your Menu Bar
+site: https://github.com/joshghent/gifbar
+tags:
+- react
+- javascript
+upforgrabs:
+  name: help wanted
+  link: https://github.com/joshghent/gifbar/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```

Comment:
@joshghent Please change this to https://github.com/joshghent/gifbar/labels/help%20wanted

Reply:
Hey, no worries! Just done this, thanks for reviewing 👍